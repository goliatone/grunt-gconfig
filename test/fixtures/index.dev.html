<!DOCTYPE html>
<html>
<head>
	<title>Test GConfig grunt task: Dev Environment</title>
</head>
<body>
</body>
</html>