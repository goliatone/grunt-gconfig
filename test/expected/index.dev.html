<!DOCTYPE html>
<html>
<head>
	<title>Test GConfig grunt task: Dev Environment</title>
<meta name="platform:ip" content="localhost">
<meta name="platform:port" content="9000">
<meta name="app:endpoint" content="ws://{{ip}}:{{port}}">
<meta name="gconfigMergeGrunt:foo" content="bar">
<meta name="gconfigMergeGrunt:baz" content="fuz">
<meta name="mergedfile:path" content="fixtures/gconfig.merge.json"></head>
<body>
</body>
</html>